src/components/transactions/Warnings/ParaswapErrorDisplay.tsx

```diff
@@ -0,0 +1,31 @@
+import { Trans } from '@lingui/macro';
+import { Box, Typography } from '@mui/material';
+import { Warning } from 'src/components/primitives/Warning';
+import { TxErrorType } from 'src/ui-config/errorMapping';
+
+import { GasEstimationError } from '../FlowCommons/GasEstimationError';
+
+const USER_DENIED_SIGNATURE = 'MetaMask Message Signature: User denied message signature.';
+const USER_DENIED_TRANSACTION = 'MetaMask Message Signature: User denied message signature.';
```

Comment:
should this be wrapped in a trans?

Reply:
It's not displayed so it doesn't need a Trans. These are used to check the error message returned by MetaMask to decide if the error is was from the user denying the action or gas estimation